### PR TITLE
Fix mobile view for skill search #2622

### LINKED ISSUE #2622: Mobile View: no skill example bubble
There is no skill bubble on search and cat pages in mobile view.

Please implement it.

Screenshot_2019-07-17-19-40-14-207_com android chrome

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2622

Changes: 
- Added bubble
- Fixed mobile view, skills sorting header

Demo Link: https://pr-2624-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 

Screenshot 2019-07-18 at 3 13 40 AM
Screenshot 2019-07-18 at 3 13 47 AM

### CODE REVIEW

**Comment by fragm3** [comment]
@harsh-jindal updated

**Comment by harsh-jindal** [comment]
@fragm3 
Do we need to keep the search icon to the left ? Can't we have it at on the right side ?
Fix this:
Screenshot 2019-07-18 at 11 41 03 AM

**Comment by fragm3** [comment]
@harsh-jindal not aiming for so low width viewports. I think search is better on left than right, on right there are too many icons already

File: src/components/cms/SkillCardList/SkillCardList.js [hunk 1]
```diff
@@ -93,17 +102,19 @@ const Content = styled.div`
...
 `;
 
 const ImageContainerMobile = styled.div`
   padding: 0.625rem;
-  height: 7.5rem;
-  display: flex;
-  align-items: center;
 `;
+//   height: 7.5rem;
+// `;
```
Comment (harsh-jindal) [inline src/components/cms/SkillCardList/SkillCardList.js]: Remove comments

**Comment by fragm3** [comment]
@harsh-jindal done

**Comment by fragm3** [comment]
@akshatnitd If I place the bubble on right, it would look something like this:
Screenshot 2019-07-18 at 7 39 30 PM

Initially, I had thought of placing it right, but the example bubbles are of variable width and could be multiple, we can show more example if we place them at the bottom. Also, rating container can take more width. It would be tough to adjust bubbles horizontally but vertically there's more space.

Properly created skills would be having more than one bubble

**Comment by fragm3** [comment]
@akshatnitd made the changes, looking better now:
Screenshot 2019-07-18 at 7 52 14 PM

**Pull request merged**